cmake/onnxruntime_unittests.cmake

```diff
@@ -396,18 +396,6 @@ endif()
 
 set (onnxruntime_test_providers_dependencies ${onnxruntime_EXTERNAL_DEPENDENCIES})
 
-if(onnxruntime_USE_CUDA)
-  list(APPEND onnxruntime_test_providers_dependencies onnxruntime_providers_cuda)
-endif()
-
-if(onnxruntime_USE_DNNL)
```

Comment:
why these changes for DNNL and OPENVINO ?

Reply:
I noticed these extraneous dependencies were here, so removed them for the other shared providers too.